pkg/watch/registrar.go

```diff
@@ -113,7 +114,7 @@ func (r *recordKeeper) Update(parentName string, m vitalsByGVK) {
 
 // ReplaceRegistrarRoster replaces the desired set of watches for the specified registrar using provided roster.
 // Ownership is taken over roster - it is not currently deep-copied.
-func (r *recordKeeper) ReplaceRegistrarRoster(reg *Registrar, roster map[schema.GroupVersionKind]vitals) {
+func (r *recordKeeper) ReplaceRegistrarRoster(ctx context.Context, reg *Registrar, roster map[schema.GroupVersionKind]vitals) {
```

Comment:
It looks like this context is no longer used?

Reply:
Done.